Terraform/terraform-deployment.yml

```diff
@@ -0,0 +1,61 @@
+name: "Terraform Deploy"
+on:
+  push:
+    branches:
+      - master
+
+env:
+  TF_VER: 0.12.29
```

Comment:
Is it possible to say LATEST? Otherwise the template might require update every time a newer version is available.

Reply:
makes sense. If not provided, terraform_version field in the action defaults to 'latest'. @manojvazirani - Can you please clarify if there is any specific intent behind using the version in the template?

Reply:
That is an interesting point. Thanks @N-Usha for the tip.